fe1-web/store/Storage.ts

```diff
@@ -45,7 +37,9 @@ export function storeInit(): PersistStoreConfig {
     // clears the local cached storage as well as the state of the storage
     let newState = state;
     if (action.type === 'CLEAR_STORAGE') {
-      storage.removeItem('persist:root'); newState = undefined;
+      // unsafe, asynchronous operation:
+      AsyncStorage.removeItem('persist:root');
```

Comment:
Should we create an adapter for the asynchronous storage for the students? 

I don't exactly see yet how the components will update their state/view with our new network layer (aka. after handling the server answer) so it might not be useful at all.

Reply:
As I see it, the redux store will take care of managing everything, so there's no need.

Furthermore, this code is unused and only available as "debug" functionality to reset the entire system. I think it's okay to leave it with a warning.